.spelling

```diff
@@ -479,6 +479,54 @@ embeddable
 p99
 vCPU
 AES-NI
+3.99ms
+72.96ms
+18x
+8x
+appswitch
+appswitch-0
+AppSwitch
+bitpipe
+DaemonSet
+datapath
+decap
+decapsulated
+delayerification
+Delayerification
+Devirtualization
```

Comment:
this is a compiler concept. Using it in context of networking will throw people off. Also suggest using proper phrase instead of new de-* terms